### PR TITLE
Feature full edit fields

### LINKED ISSUE #40: Distinguish purchased and manufactured parts in full edit page
Make sure to integrate with bootstrap beautification. (Look at switch in bootstrap)
**Time Estimate**: 1 day

**In Progress**
**Time So Far**: 3 hours

### PR DESCRIPTION
**Pull request by iangallagherm**
Add ordering status and vendor fields to full edit page.

### CODE REVIEW

File: frontend/src/PartEdit.js [hunk 1]
```diff
@@ -63,6 +78,7 @@ function PartEdit(props) {
               <p>{part.subsystem}</p>
             </Col>
           </Form.Group>
+
```
Comment (Rockjack00) [inline frontend/src/PartEdit.js:81]: Why newlines tho...

File: frontend/src/PartEdit.js [hunk 2]
```diff
@@ -139,7 +170,7 @@ function PartEdit(props) {
                   <Form.Group>
                     <Form.Label>Available</Form.Label>
                     <Form.Control
-                      className="catalog-entry__number catalog-entry__number--available"
+                      className="full-edit__number full-edit__number--available"
```
Comment (Rockjack00) [inline frontend/src/PartEdit.js]: remove duplicate class name

File: frontend/src/Select.js [hunk 3]
```diff
@@ -66,7 +66,6 @@ function OrderingPrioritySelect(props) {
     },
   };
 
-  // TODO: turn this into a pill :O
```
Comment (Rockjack00) [inline frontend/src/Select.js:69]: Lol thanks

File: frontend/src/PartEdit.js [hunk 4]
```diff
@@ -28,12 +32,23 @@ function PartEdit(props) {
...
+      case "ordering_status":
+        value = serializeType("Status", value);
+        break;
+      default:
+        break;
     }
 
     if (event.target.type === "number") {
-      value = parseInt(event.target.value);
+      if (name === "vendor_cost") {
```
Comment (rahul-go) [inline frontend/src/PartEdit.js:47]: Could be a ternary operator.

File: frontend/src/Select.js [hunk 5]
```diff
@@ -80,4 +79,49 @@ function OrderingPrioritySelect(props) {
   );
 }
 
-export { Select as default, SubassemblySelect, OrderingPrioritySelect };
+function OrderingStatusSelect(props) {
+  const value = props.value ? parseType("Status", props.value._type) : "";
+
+  // Option: style
+  const statuses = {
```
Comment (Rockjack00) [inline frontend/src/Select.js:86]: Nit: We should probably refactor these style dictionaries out eventually, but not important now.

**Review by Rockjack00 — CHANGES_REQUESTED** [review]
See comments

**Review by rahul-go — COMMENTED** [review]
Looks good to me. I'll let Tyler approve since he set up the Bootstrap stuff.

File: frontend/src/Select.js (hunk 5 above)
Comment (iangallagherm) [reply frontend/src/Select.js:86]: Agreed and agreed.

**Review by Rockjack00 — APPROVED** [review]
Good.

**Pull request merged**